extensions/in-memory/assetindex-memory/src/main/java/org/eclipse/dataspaceconnector/metadata/memory/InMemoryAssetIndexSeeder.java

```diff
@@ -0,0 +1,17 @@
+package org.eclipse.dataspaceconnector.metadata.memory;
+
+import org.eclipse.dataspaceconnector.spi.types.domain.asset.Asset;
+import org.eclipse.dataspaceconnector.spi.types.domain.transfer.DataAddress;
+
+public class InMemoryAssetIndexSeeder {
```

Comment:
I'm more sure whether I understood the intention. The idea is that other extensions reference the in memory asset index extension, create a new seeder, and. use it to add data to the asset index?

Reply:
Thanks for your answer. Maybe you can help me here. Is it ok for low level extensions to reference each other? I always thought dependencies between extensions must be handled using the SPI and ServiceExtensionContext.